src/Models/FileManagement/FileManagementVariantLogData.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace GoApptiv\FileManagement\Models\FileManagement;
+
+class FileManagementVariantLogData
```

Comment:
Kindly create migration class in model folders.

Reply:
Done.